ceph_salt/core.py

```diff
@@ -44,18 +45,21 @@ def ipsv6(self):
 
     @property
     def public_ip(self):
+        def _is_loopback(addr):
+            return ipaddress.ip_address(addr).is_loopback
+
         if self._public_ip is None:
             result = GrainsManager.get_grain(self.minion_id, 'fqdn_ip4')
             _public_ip = result[self.minion_id][0]
-            if _public_ip == '127.0.0.1':
-                logger.debug('fqdn_ipv4 grain is 127.0.0.1, falling back to ipv4 grain')
```

Comment:
For better debugging I guess it is better to print the _public_ip here the same way as it was before

Reply:
added `_public_ip` as part of the log message